bundles/config/org.eclipse.smarthome.config.core.test/META-INF/MANIFEST.MF

```diff
@@ -14,6 +14,7 @@ Import-Package:
  org.eclipse.smarthome.test,
  org.eclipse.smarthome.test.java,
  org.hamcrest;core=split,
+ org.hamcrest.core,
```

Comment:
Is that normal to have "org.hamcrest;core=split," and "org.hamcrest.core,"?

Reply:
Yes, two different statements.
@kaikreuzer ready to get merged.